StRoot/StFstClusterMaker/StFstClusterMaker.cxx

```diff
@@ -0,0 +1,111 @@
+#include <climits>
+
+#include "StFstClusterMaker.h"
+#include "StEvent.h"
+#include "StEvent/StEnumerations.h"
+#include "StFstUtil/StFstCollection.h"
+#include "StFstUtil/StFstRawHit.h"
+#include "StFstUtil/StFstRawHitCollection.h"
+#include "StFstUtil/StFstCluster.h"
+#include "StFstUtil/StFstClusterCollection.h"
+#include "StEvent/StFstConsts.h"
+#include "StFstClusterMaker/StFstIClusterAlgo.h"
+#include "StFstClusterMaker/StFstScanRadiusClusterAlgo.h"
+StFstClusterMaker::StFstClusterMaker( const char *name ) : StMaker(name), mFstCollectionPtr(0), mClusterAlgoPtr(0), mTimeBin(UCHAR_MAX), mSplitCluster(true)
+{
+   /* nothing to do */
+};
+
+StFstClusterMaker::~StFstClusterMaker()
+{
```

Comment:
Not sure about this comment. obsolete statement?

Reply:
I see now that you were referring to the comment (question):
//Is deleted as part of structure(VP)   delete mFstCollectionPtr;

I am investigating before I simply remove that - dont want memory leaks